api/v1beta1/hyperconverged_types.go

```diff
@@ -856,6 +856,22 @@ type NodeInfoStatus struct {
 	WorkloadsArchitectures []string `json:"workloadsArchitectures,omitempty"`
 	// ControlPlaneArchitectures is a distinct list of the CPU architecture of the control-plane nodes.
 	ControlPlaneArchitectures []string `json:"controlPlaneArchitectures,omitempty"`
+	// RecommendedCpuModels is a list of recommended CPU models for the cluster based on available nodes
+	RecommendedCpuModels []CpuModelInfo `json:"recommendedCpuModels,omitempty"`
```

Comment:
Please add list type and key. This will ensure that we only have one object for each CPU name:

```suggestion
	// +listType=map
	// +listMapKey=name
	RecommendedCpuModels []CpuModelInfo `json:"recommendedCpuModels,omitempty"`
```